src/cli/dsc/schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Oh My Posh DSC State",
+  "description": "State configuration for Oh My Posh DSC (Desired State Configuration)",
+  "properties": {
+    "configurations": {
+      "type": "array",
+      "title": "Configuration List",
+      "description": "List of Oh My Posh configurations",
+      "items": {
+        "type": "object",
+        "properties": {
+          "path": {
+            "type": "string",
+            "title": "Configuration Path",
+            "description": "File path to the Oh My Posh configuration file"
+          },
+          "format": {
+            "type": "string",
+            "title": "Configuration Format",
+            "description": "Format of the configuration file",
+            "enum": [
+              "json",
+              "yaml",
+              "toml"
+            ]
+          },
+          "data": {
+            "title": "Configuration Data",
+            "description": "The actual Oh My Posh configuration object",
+            "$ref": "https://ohmyposh.dev/schema.json"
```

Comment:
This isn't necessarily an issue, but it's best practice as part of your build step to ensure you have a canonically bundled schema for your resource, so you'll want the data schema added to the `$defs` keyword.